src/components/Content/ResearchBounty.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    class="content-block p-1 d-flex flex-column flex-md-row align-items-center justify-content-between"
+  >
+
+    <div>
+      <span class="mr-1 bounty-desc">Items needed:</span>
+      <span>TODO Items go here</span>
```

Comment:
Note to self: I'm passing a list like {money: 1, itemIdExample: 5} to this and want to parse it and display an image of the item and the count needed on the UI like
[MoneyIcon]: 1
[itemIcon]: 5
An example of how to do this might be the cargo shop UI